nativelib/src/main/resources/gc/nogc/gc.c

```diff
@@ -0,0 +1,40 @@
+#include <gc.h>
+#include "gc.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+// Dummy GC that allocates memory in 1G chunks and never frees.
+
+void* start = 0;
+void* last = 0;
+
+#define CHUNK (1024*1024*1024)
```

Comment:
We should also make this smaller. It was good for benchmarks to completely remove GC out of question, but for actual apps we need finer-grain granularity. Lets say `4M` for example. Any allocations above that amount should go directly to `malloc`, sidestepping bump allocation.